Java8/src/ro/siit/App.java

```diff
@@ -0,0 +1,76 @@
+package ro.siit;
+
+/**
+ * @author Georgiana
+ * @version 1.0
+ * @sience 08/09/2019
+ * Main class
+ *
+ */
+
+import ro.siit.entities.Person;
+import java.io.*;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import static ro.siit.readerwriter.Reader.read;
+import static ro.siit.readerwriter.Writer.writePerson;
+
+public class App {
+
+    public static void main(String[] args) throws ParseException {
+        /**
+         * Create a new list to save the person objects
+         */
+        List<Person> pers = new ArrayList<>();
+        pers.add(new Person("Ioana", "Constantinescu", "1992/11/22"));
+        pers.add(new Person("Adriana", "Popescu", "1982/11/28"));
+        pers.add(new Person("Oana", "Iliescu", "1992/03/15"));
+        pers.add(new Person("Bogdan", "Georgescu", "1995/03/12"));
+        pers.add(new Person("Bianca", "Enescu", "1981/11/12"));
+
+
+        try {
+            /**
+             * Writing objects in  a new File - input.txt
+             */
+            writePerson(pers, "input.txt");
+
+
+            /**
+             * Read objects from the input.txt
+             */
+            List<Person> persList = new ArrayList<>();
+            read(persList, "input.txt");
```

Comment:
Instead of passing an empty `List` to the `read()` method, you could have returned the list and assigned it to `persList` like this:
`List<Person> persList = read("input.txt");`
It's cleaner and easier to see what's happening in the `read()` method. It's also easier to test. Besides, the returned `File` is not used anyway.